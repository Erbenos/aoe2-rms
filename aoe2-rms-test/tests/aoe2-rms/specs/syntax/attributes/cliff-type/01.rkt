#lang aoe2-rms
; Make the cliffs brown to match dirt and desert terrains.

<CLIFF-GENERATION>
(cliff-type 'CT_DESERT)
