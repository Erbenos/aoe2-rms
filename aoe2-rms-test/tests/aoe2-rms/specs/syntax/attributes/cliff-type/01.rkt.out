<CLIFF_GENERATION>
cliff_type CT_DESERT
